lib/companies_web/views/company_view.ex

```diff
@@ -28,4 +28,15 @@ defmodule CompaniesWeb.CompanyView do
   def select_industries(industries) do
     Enum.map(industries, fn %{id: id, name: name} -> {name, id} end)
   end
+
+  def selected_text(%{}, _), do: ""
+  def selected_text(%{search: %{}}, _), do: ""
```

Comment:
Do we need these if they both return `""`? Could we just put `""` where you call these?

If for some reason we do need this function, we could simplify to one I believe:
```elixir
def selected_text(_, _), do: ""
```